### DISCUSSION TITLE
[A-L] (2023/24) Foglio 1 - Esercizio 3

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#158] [Esercizi - Viaggi/Piazza] [answered]
![es 3](https://github.com/Elia-Belli/algebra2324/assets/107944476/3249b41a-14b0-48ab-b611-7076842e4fff)

**Comment by Elia-Belli** [comment, marked as answer]
# Soluzione del Prof. Viaggi
$(a,b) \in (R_1 \cap ... \cap R_n)^{-1}$
$\iff (b,a) \in (R_1 \cap ... \cap R_n) $
$\iff (b,a) \in R_j, \forall j \leq n$
$\iff (a,b) \in R_j^{-1}, \forall j \leq n$
$\iff (a,b) \in R_1^{-1} \cap ... R_n^{-1}$

**Comment by FeddyLix17** [comment]
Siano tutti gli elementi $\Large a,\ b \in A\ |\ (a,\ b) \in R_1 \cap \dots \cap R_n$ ovvero 

$$
    \Large \forall (a,b) \in R_i \quad 1 \leq i \leq n
$$

allora risulterà che ogni coppia $\Large (b,\ a) \in (R_1 \cap \dots \cap R_n)^{-1}$, ovvero 

$$
    \Large \forall (b,a) \in R_i^{-1} \quad 1 \leq i \leq n
$$
